test/gui/shared/scripts/bdd_hooks.py

```diff
@@ -113,7 +116,10 @@ def hook(context):
         os.makedirs(tmp_dir)
 
     # sync connection folder display name
-    set_config("syncConnectionName", "Personal" if get_config("ocis") else "ownCloud")
+    set_config(
+        "syncConnectionName",
+        "My Folder" if get_config("predefined_users") else "Personal",
```

Comment:
Coule we make this configurable via config as well? Something like `PERSONAL_SYNC_FOLDER=`. default can be `Personal`